packages/components/src/radio/stories/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Internal dependencies
+ */
+import RadioGroup from '../../radio-group';
+import Radio from '../';
+
+export default { title: 'Components/Radio', component: Radio };
+
+// Radio components can be in their own component, but must be a descendent of
+// a RadioGroup component at some point.
+const RadioOptions = () => {
+	/* eslint-disable no-restricted-syntax */
+	// id is required for server side rendering
+	return (
+		<>
+			<Radio id="option1-radio" value="option1">
+				Option 1
+			</Radio>
+			<Radio id="option2-radio" value="option2">
+				Option 2
+			</Radio>
+		</>
+	);
+	/* eslint-enable no-restricted-syntax */
+};
```

Comment:
I'd move this function into the `_default` function below. Otherwise Storybook won't show the encapsulated code:
![image](https://user-images.githubusercontent.com/3068563/78413895-eb16e080-75ef-11ea-8e78-a880d84dad5d.png)

Also, I guess the same suggestion on IDs applies here. :)